pkg/deploymanager/storage_system.go

```diff
@@ -1,13 +1,34 @@
 package deploymanager
 
 import (
+	"fmt"
+	"time"
+
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
 	odfv1alpha1 "github.com/red-hat-storage/odf-operator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	utilwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
-// GetDefaultStorageSystem returns the default storagesystem
-func GetDefaultStorageSystem() *odfv1alpha1.StorageSystem {
+// GetDefaultStorageCluster returns the default StorageCluster
+func GetDefaultStorageCluster() *ocsv1.StorageCluster {
+	return &ocsv1.StorageCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "ocs-storagecluster",
+			Namespace: InstallNamespace,
+		},
+		Spec: ocsv1.StorageClusterSpec{
+			Version: "0.0.0.0.0",
```

Comment:
Can you add a comment here on why are we doing this. We do remember it now but later we may forget about it.

Reply:
Done Now